Sources/JWTKit/Claims/AudienceClaim.swift

```diff
@@ -11,10 +11,83 @@
 /// Use of this claim is OPTIONAL.
 public struct AudienceClaim: JWTClaim, Equatable, ExpressibleByStringLiteral {
     /// See `JWTClaim`.
-    public var value: String
+    public var value: [String]
 
     /// See `JWTClaim`.
-    public init(value: String) {
+    public init(value: [String]) {
+        precondition(!value.isEmpty, "An audience claim must have at least one audience.")
         self.value = value
     }
+    
+    /// Convenience for the almost universal case of only a single audience.
+    public init(value: String) {
+        self.init(value: [value])
+    }
+    
+    /// See `ExpressibleByStringLiteral`.
+    public init(stringLiteral value: String) {
+        self.init(value: value)
+    }
+    
+    /// Verify that the given audience is included as one of the claim's
+    /// intended audiences by simple string comparison.
+    public func verifyIntendedAudience(includes audience: String) throws {
```

Comment:
Should this be an `includes oneOf: String...` type method instead?  Would there ever be a need to ensure that something was intended for multiple audiences?  I'm guessing no, but this seems like the right time to ask.

Reply:
Ignore that, it would just make the API confusing and be a very rare case.